packages/sitebuilder/src/wizards/first-time-configuration/screens/ContentTone.tsx

```diff
@@ -1,87 +1,84 @@
 	};
 
-	// useEffect(() => {
-	// 	if (! form) {
-	// 		return;
-	// 	}
-	// 	shouldBlockNextStep(
-	// 		! form.siteDescription.value ||
-	// 			! form.customerDescription.value ||
-	// 			! form.valueProposition.value ||
-	// 			! form.productsAndServices.value,
-	// 		2
-	// 	);
-	// }, [form]);
+	const handleChange = (prop: keyof FtcFormItemsInterface) => (event: SelectChangeEvent<any>) => {
+		setFormValue(prop, event.target.value);
+	};
+
+
+	useEffect(() => {
+		if (! form) {
+			return;
+		}
+		shouldBlockNextStep(! form.siteKeywords.value || ! form.sitePersonality.value);
+	}, [form]);
 
 	return (
 		<PageWrapper>
 			<Form>
-				<Stack spacing={ 2 }>
-					<div>
+				<Stack spacing={ 4 }>
+					<Box sx={ { display: 'flex', flexDirection: 'column', gap: 1 } }>
 						<Typography variant="h3">{ keywords.title }</Typography>
-						<Typography variant="body2" sx={ { color: 'text.secondary', marginTop: '8px' } }>{ keywords.description }</Typography>
-					</div>
-					<FormField
-						field={
-							<ChipInput
-								rows={ 3 }
-								selectedTags={ updateKeywords }
-								placeholder={
-									keywords.placeholder
-								}
-								required
-								tags={ form.siteKeywords.value }
-							/>
-						}
-						helperText={ keywords.helperText }
-					/>
-					{ /* <FormField
-						field={
-							<SelectInput
-								fullWidth
-								onChange={ handleInputChange('sitePersonality') }
-								value={ form.sitePersonality.value }
-								placeholder={
-									industryDetails.personalityPlaceholder
-								}
-							>
-								{ personalityOptions.map((item) => (
-									<MenuItem id={ item } key={ item } value={ item }>
-										{ item }
-									</MenuItem>
-								)) }
-							</SelectInput>
-						}
-						label={ industryDetails.personalityLabel }
-					/> */ }
+						<Typography variant="body2" sx={ { color: 'text.secondary', marginBottom: '8x' } }>{ keywords.description }</Typography>
+						<FormField
+							field={
+								<ChipInput
+									rows={ 3 }
+									selectedTags={ updateKeywords }
+									placeholder={
+										keywords.placeholder
+									}
+									required
+									tags={ form.siteKeywords.value }
+								/>
+							}
+							helperText={ keywords.helperText }
+						/>
+					</Box>
+					<Box sx={ { display: 'flex', flexDirection: 'column', gap: 1 } }>
+						<Typography variant="h3">{ contentPersonality.title }</Typography>
+						<Typography variant="body2" sx={ { color: 'text.secondary', marginBottom: '8px' } }>{ contentPersonality.description }</Typography>
+						<FormField
+							field={
+								<SelectInput
+									fullWidth
+									sx={ { maxWidth: '400px' } }
+									onChange={ handleChange('sitePersonality') }
+									value={ form.sitePersonality.value }
+								>
+									{ personalityOptions.map((opt) => (
+										<MenuItem key={ opt.name } value={ opt.name }>
+											{ opt.name }
+										</MenuItem>
+									)) }
```

Comment:
Please update `value` attribute to use the values expected by Prophecy (see my note in `data/first-time-configuration.ts`)